src/components/Graph.react.js

```diff
@@ -108,6 +108,20 @@ PlotlyGraph.propTypes = {
      */
     id: PropTypes.string,
 
+    /**
+     * If True, the Plotly.js plot will be fully responsive to window resize
+     * and parent element resize event. This is achieved by overriding
+     * `config.responsive` to True, `figure.layout.autosize` to True and unsetting
+     * `figure.layout.height` and `figure.layout.width`.
+     * If False (default), the Plotly.js plot may exhibit certain responsive
+     * behaviors on its own but nothing is done in Dash to help this behavior.
+     * This is the legacy behavior of the Graph component.
+     * If 'auto', the Graph will determine if the Plotly.js plot can be made fully
+     * responsive (True) or not (False) based on the values in `config.responsive`,
+     * `figure.layout.autosize`, `figure.layout.height`, `figure.layout.width`.
+     */
+    responsive: PropTypes.oneOf([true, false, 'auto']),
```

Comment:
Allow users to define the behavior they want for their graph, allowing them to override the normal behavior for their `config`/`figure` values.